sphinx/directives/other.py

```diff
@@ -39,6 +39,7 @@ class TocTree(Directive):
     final_argument_whitespace = False
     option_spec = {
         'maxdepth': int,
+        'name': str,
```

Comment:
Works for me. I don't really care about the implementation as much as just
being able to get it work as a user :)

On Fri, Mar 6, 2015 at 8:20 PM, Takayuki SHIMIZUKAWA <
notifications@github.com> wrote:

> In sphinx/directives/other.py
> <https://github.com/sphinx-doc/sphinx/pull/1700#discussion_r25993164>:
>
> > @@ -39,6 +39,7 @@ class TocTree(Directive):
> >      final_argument_whitespace = False
> >      option_spec = {
> >          'maxdepth': int,
> > +        'name': str,
>
> @ericholscher <https://github.com/ericholscher> I guess it's time to
> release 1.3. How do you think about :caption:?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sphinx-doc/sphinx/pull/1700/files#r25993164>.
>



-- 
Eric Holscher
Maker of the internet residing in Portland, Or
http://ericholscher.com